lib/routes/withdraw/reverse_swap/in_progress/reverse_swap_in_progress.dart

```diff
@@ -46,19 +46,20 @@ class _TxLink extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final text = context.texts();
+    final mempoolHelper = MempoolHelper();
```

Comment:
Better create it outside build method

Reply:
it is not a problem because we are getting an existing instance not creating a new instance

Reply:
Note sure how this might cause a problem, could you expand on hos this might be an issue.